### PR TITLE
Floating point comparison

### PR DESCRIPTION
**Pull request by rdaly525**
Dependent on #81 

Uses:
https://community.arm.com/developer/ip-products/processors/b/processors-ip-blog/posts/condition-codes-4-floating-point-comparisons-using-vfp

### CODE REVIEW

**Comment by rdaly525** [comment]
@leonardt, the rtl generation seems to be complaining about an indent error but if I comment out the rtl tests, functional model works fine. Could you take a look?

File: lassen/asm.py [hunk 1]
```diff
@@ -73,9 +73,30 @@ def fgetmant ():
...
+    return fp_cmp(Cond.SGT)
+
+def fp_ge():
+    return fp_cmp(Cond.SGE)
+
+def fp_lt():
+    return fp_cmp(Cond.N)
+
+def fp_le():
+    return fp_cmp(Cond.ULE)
```
Comment (phanrahan) [inline lassen/asm.py]: Should this be SLE?

**Comment by rdaly525** [comment]
@phanrahan, thoughts on this? I am open to other approaches although this has the benefit of being consistent with ARM.

**Comment by Kuree** [comment]
Can this be merged or requires more tests/refactoring?

**Comment by rdaly525** [comment]
Closing this in favor of #99

**Pull request closed without merging**